src/foolscap/test/test_connection.py

```diff
@@ -49,18 +49,18 @@ def checkUnknownEndpoint(self, hint):
         self.failureResultOf(d, ipb.InvalidHintError)
 
     def testConvertLegacyHint(self):
-        self.failUnlessEqual(tcp.convert_legacy_hint("127.0.0.1:9900"),
+        self.assertEqual(tcp.convert_legacy_hint("127.0.0.1:9900"),
                              "tcp:127.0.0.1:9900")
-        self.failUnlessEqual(tcp.convert_legacy_hint("tcp:127.0.0.1:9900"),
+        self.assertEqual(tcp.convert_legacy_hint("tcp:127.0.0.1:9900"),
                              "tcp:127.0.0.1:9900")
-        self.failUnlessEqual(tcp.convert_legacy_hint("other:127.0.0.1:9900"),
+        self.assertEqual(tcp.convert_legacy_hint("other:127.0.0.1:9900"),
                              "other:127.0.0.1:9900")
         # this is unfortunate
-        self.failUnlessEqual(tcp.convert_legacy_hint("unix:1"), "tcp:unix:1")
+        self.assertEqual(tcp.convert_legacy_hint("unix:1"), "tcp:unix:1")
         # so new hints should do one of these:
-        self.failUnlessEqual(tcp.convert_legacy_hint("tor:host:1234"),
+        self.assertEqual(tcp.convert_legacy_hint("tor:host:1234"),
                              "tor:host:1234") # multiple colons
-        self.failUnlessEqual(tcp.convert_legacy_hint("unix:fd=1"),
+        self.assertEqual(tcp.convert_legacy_hint("unix:fd=1"),
                              "unix:fd=1") # equals signs, key=value -style
```

Comment:
These changes destroyed the hanging indent.. does style matter for this, or is there an auto-formatter than's known to work over foolscap?

This isn't the only file affected but it's the only place I'll leave a comment.

Reply:
What is the hanging indent ?

Reply:
Previously, the `(` was aligned with the indent of the next line:
```
        self.failUnlessEqual(tcp.convert_legacy_hint("unix:fd=1"),
                             "unix:fd=1") # equals signs, key=value -style
```
Now, it isn't:
```
        self.assertEqual(tcp.convert_legacy_hint("unix:fd=1"),
                             "unix:fd=1") # equals signs, key=value -style
```

Reply:
```

Reply:
Ok got you.

Reply:
Fixed.